sql-db/vertx-sql/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>vertx-sql</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: SQL exploratory testing</name>
```

Comment:
This name does not sound good to me, it should be <name>Quarkus QE TS: SQL Database: Vertx SQL</name>

Reply:
What are the rules for these names?

Reply:
This change is remaining

Reply:
@Sgitario what are the rules for these names?

Reply:
Please, have a look to the rest of the SQL-DB modules and follow the same name convention....

Reply:
Changed the name